src/js/stores/MesosSummaryStore.js

```diff
@@ -178,6 +178,42 @@ class MesosSummaryStore extends GetSetBaseStore {
     });
   }
 
+  getFrameworkTasksByName(serviceName = '') {
```

Comment:
And could we call it something with `count` in it to make it obvious what is to be expected returned?